### PR TITLE
Biomarker details

### PR DESCRIPTION
**Pull request by anbadrin**

### CODE REVIEW

**Review by axdanbol — CHANGES_REQUESTED** [review]

File: libs/components/atoms/src/index.ts [hunk 1]
```diff
@@ -4,5 +4,5 @@ export * from './lib/gradient-legend/gradient-legend.component';
 export * from './lib/label-box/label-box.component';
 export * from './lib/required-textbox/required-textbox.component';
 export * from './lib/size-legend/size-legend.component';
-
+export * from './lib/required-textbox/required-textbox.component';
```
Comment (axdanbol) [inline libs/components/atoms/src/index.ts:7]: Duplicate export

File: libs/components/behavioral/src/lib/biomarker-details/biomarker-details.component.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    justify-content: space-between;
+    align-items: center;
+    grid-column: 1 / 4;
+    grid-row: 1;
+    padding: 0.25rem 2rem;
+    gap: 0.5rem;
+  }
+
+  .table {
+    grid-column: 1 / 4;
```
Comment (axdanbol) [inline libs/components/behavioral/src/lib/biomarker-details/biomarker-details.component.scss:18]: Consider replacing grid-column/grid-row with grid-area and add a grid-template-area on the :host level for readability
https://css-tricks.com/snippets/css/complete-guide-grid/#aa-grid-template-areas

File: libs/components/organisms/src/lib/biomarker-table/biomarker-table.component.html [hunk 3]
```diff
@@ -1,40 +1,44 @@
-<table class="table" mat-table [dataSource]="dataSource">
-  <caption class="sr-only">
-    Cell type table by Biomarker
-  </caption>
+<div class="table-wrapper">
```
Comment (axdanbol) [inline libs/components/organisms/src/lib/biomarker-table/biomarker-table.component.html:1]: Why do you need all these wrappers divs!?

File: libs/components/organisms/src/lib/biomarker-table/biomarker-table.component.scss [hunk 4]
```diff
@@ -9,10 +9,10 @@
   }
 
   .table {
-    overflow-x: auto;
-    overflow-y: auto;
     min-width: auto;
+    min-height: auto;
     font-size: 0.875rem;
+    position: absolute;
```
Comment (axdanbol) [inline libs/components/organisms/src/lib/biomarker-table/biomarker-table.component.scss:15]: This seems sketchy...

**Review by axdanbol — CHANGES_REQUESTED** [review]

File: libs/components/organisms/src/lib/biomarker-table/biomarker-table.component.scss (hunk 4 above)
Comment (adityarudrawar) [reply libs/components/organisms/src/lib/biomarker-table/biomarker-table.component.scss:16]: If we dont use absolute positioning then the scroll bar on the right hand side, and bottom scroll bar will move to the end of the screen in their respective directions.

**Review by axdanbol — APPROVED** [review]

**Pull request merged**